src/Interactive/Host/Interactive/Core/InteractiveHost.cs

```diff
@@ -37,9 +40,7 @@ internal sealed partial class InteractiveHost : MarshalByRefObject
 
         private LazyRemoteService _lazyRemoteService;
         private int _remoteServiceInstanceId;
-
-        // Remoting channel to communicate with the remote service.
-        private IpcServerChannel _serverChannel;
+        private NamedPipeClientStream _clientStream;
```

Comment:
>private NamedPipeClientStream _clientStream; [](start = 8, length = 44)

This field shouldn't be needed as we store the jsonRpc on the remote service object.